obspy/io/sac/sactrace.py

```diff
@@ -1243,7 +1247,7 @@ def from_obspy_trace(cls, trace, keep_sac_header=True):
 
         return sac
 
-    def to_obspy_trace(self, debug_headers=False):
+    def to_obspy_trace(self, encoding='ASCII', debug_headers=False):
```

Comment:
I realize it doesn't change the code execution, but for consistency please also move `encoding` to the end here.